src/Analyzer.PowerShellRuleEngine/PowerShellRuleEngine.cs

```diff
@@ -115,13 +118,16 @@ public IEnumerable<IEvaluation> AnalyzeTemplate(TemplateContext templateContext)
                     },
                     Execution = new ExecutionOption
                     {
-                        NotProcessedWarning = false,
-
+                        UnprocessedObject = ExecutionActionPreference.Ignore,
                         // PSRule internally creates a PowerShell initial state with InitialSessionState.CreateDefault().
                         // SessionState.Minimal causes PSRule to use CreateDefault2 instead of CreateDefault.
-                        InitialSessionState = PSRule.Configuration.SessionState.Minimal
+                        InitialSessionState = SessionState.Minimal
                     }
                 };
+
+                // placeholder value for location is westus2
+                optionsForFileAnalysis.Configuration["AZURE_RESOURCE_ALLOWED_LOCATIONS"] = new[] { "westus2" };
```

Comment:
Is there any reason to leave this here?

Reply:
Yes, this changes the PSRule validation behavior to expect westus2 as an expected region, where the default is eastus. Without this we would get the AZR-000167 false positive as before.